README.md

```diff
@@ -1,38 +1,131 @@
+<br />
+
+<h3 align="center">NakVaksin</h3>
+
+<p align="center">
+A simple, easier way to get notified on your appointment.
+<br />
+<br />
+<a href="https://nakvaksin.vercel.app/">View The App</a>
+·
+<a href="https://github.com/nubpro/nakvaksin/issues">Report Bug</a>
+·
+<a href="https://github.com/nubpro/nakvaksin/issues">Request Feature</a>
+</p>
+
+<!-- TABLE OF CONTENTS -->
+<details open="open">
+  <summary><h2 style="display: inline-block">Table of Contents</h2></summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+      <ul>
+        <li><a href="#built-with">Built With</a></li>
+      </ul>
+    </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#installation">Installation</a></li>
+      </ul>
+    </li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#roadmap">Roadmap</a></li>
+    <li><a href="#contributing">Contributing</a></li>
+    <li><a href="#license">License</a></li>
+    <li><a href="#contact">Contact</a></li>
+  </ol>
+</details>
+
+<!-- ABOUT THE PROJECT -->
+
+## About The Project
+As some [MySejahtera](https://mysejahtera.malaysia.gov.my/intro_en/) user do not receive any notification and missed their appointment, or user ar force to check the application every day for the very same reason.
+<br>
+This application will provide a simple, easier way to get notified on user's vaccination appointment.
+
+### Built With
+
+-   [React JS](https://reactjs.org/)
+-   [Next JS](https://nextjs.org/)
+-   [AWS](https://aws.amazon.com/)
+
+<!-- GETTING STARTED -->
+
 ## Getting Started
 
-First, run the development server:
+### Prerequisites
+
+This is an example of how to list things you need to use the software and how to install them.
+
+-   npm
+-   yarn
+-   node.js v14 
```

Comment:
Not necessary v14, we are using v14 because of M1....

Reply:
Dang didn't realise haha